app/src/main/java/com/example/ithaca_transit_android_v2/NetworkUtils.kt

```diff
@@ -14,18 +13,23 @@ import okhttp3.OkHttpClient
 import okhttp3.Request
 import okhttp3.RequestBody.Companion.toRequestBody
 import org.json.JSONObject
+import java.util.concurrent.TimeUnit
 
 /*
  * NetworkUtils include all the networking calls needed
  */
 class NetworkUtils {
 
-    val client = OkHttpClient()
+    val client = OkHttpClient.Builder()
+        .connectTimeout(2, TimeUnit.MINUTES)
+        .writeTimeout(2, TimeUnit.MINUTES) // write timeout
+        .readTimeout(2, TimeUnit.MINUTES) // read timeout
```

Comment:
jw where did the 2 min come from? do we want the user to be waiting for an entire 2 min before timing out?

Reply:
It was just a place holder. I'll change it to 15 seconds.